META-INF/MANIFEST.MF

```diff
@@ -49,3 +49,4 @@ Import-Package: com.google.common.collect,
 Service-Component: OSGI-INF/*.xml
 Export-Package: org.openhab.binding.zwave
 Bundle-ActivationPolicy: lazy
+Require-Bundle: org.eclipse.smarthome.io.transport.serial
```

Comment:
To fix the build just remove this `Require-Bundle` manifest entry. Instead you should add `org.eclipse.smarthome.io.transport.serial` to the `Import-Package` entries.